tools/packaging/linux/create_rpm.sh

```diff
@@ -77,7 +77,7 @@ mv ${RPM_NAME}-${VERSION}.tar.gz ${BUILD_ROOT}/SOURCES/
 rm -rf ${WORK_DIR}
 
 echo "Creating the rpm package"
-rpmbuild --define "VERSION $VERSION" --define "RPM_NAME $RPM_NAME" --define "_topdir ${BUILD_ROOT}" -bb -v --clean ${SPEC_FILE} --target ${ARCH}-linux
+rpmbuild --define "VERSION $VERSION" --define "RPM_NAME $RPM_NAME" --define "_topdir ${BUILD_ROOT}" --define "_source_filedigest_algorithm 8" --define "_binary_filedigest_algorithm 8" -bb -v --clean ${SPEC_FILE} --target ${ARCH}-linux
```

Comment:
if customers use this script to build rpm without signing, will the rpm be able to be installed ?

Reply:
Yes, this is not related to signing itself, but related to RHEL 8.2 FIPS check. Customer can still build and run their own RPM without any issue.